agents/che-core-api-agent/src/test/resources/run_launcher_bats_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
separate PR is fine for me as this PR should unblock the usage of a specific branch of Che for OpenShift.